specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,509 @@
+        }
+    }
+}
+```
+
+## Notification Progress Updates
+
+Sometimes a developer would like to show progress bar related updates in a Notification:
+
+![Notification Progress](https://docs.microsoft.com/windows/apps/design/shell/tiles-and-notifications/images/toast-progressbar-annotated.png)
+
+To accomplish that, the developer will need to use the AppNotificationProgressData construct to
+update the progress bar on the screen.
+
+```cpp
+// Send first Notification Progress Update
+void SendUpdatableNotificationWithProgress()
+{
+    winrt::hstring tag = L"weekly-playlist";
+    winrt::hstring group = L"downloads";
+
+    winrt::hstring payload =
+        LR"(<toast launch="action = viewDownload &amp; downloadId = 9438108">
+        <visual>
+            <binding template = "ToastGeneric">
+                <text>Downloading this week's new music...</text>
+                <progress
+                    title = "{progressTitle}
+                    value = "{progressValue}"
+                    valueStringOverride = "{progressValueString}"
+                    status = "{progressStatus}" />
+            </binding>
+        </visual>
+    </toast>)";
+
+    winrt::AppNotification notification(payload);
+    notification.Tag(tag);
+    notification.Group(group);
+
+    // Assign initial values for first notification progress UI
+    winrt::AppNotificationProgressData data(1);
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.6); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"15/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+
+    notification.Progress(data);
+    winrt::AppNotificationManager::Default().Show(notification);
+}
+
+// Send subsequent progress updates
+winrt::Windows::Foundation::IAsyncAction UpdateProgressAsync()
+{
+    winrt::hstring tag = L"weekly-playlist";
+    winrt::hstring group = L"downloads";
+
+    // Assign new values
+    winrt::AppNotificationProgressData data(2 /* Sequence number */ );
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.7); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"18/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+
+    auto result = co_await winrt::AppNotificationManager::Default().UpdateAsync(data, tag, group);
+    if (result == winrt::AppNotificationProgressResult::AppNotificationNotFound)
+    {
+        LOG_HR_MSG(E_NOT_SET, "Toast Progress Update Failed since the previous notification update is missing or has been cleared by the user!");
+    }
+}
+```
+
+# Remarks
+
+## Registration
+
+The developer should always call the Notification Registration API first to register the current
```

Comment:
name the registration method instead of talking about it indirectly. 
the "The developer" is indirect, drop it.